README.md

```diff
@@ -143,11 +142,7 @@ let ma = MyArray.from([1,2,3]);
 
 RegExp subclassing machinery involves both @@species and dynamic property lookups on the `this` value in various prototype methods. Property lookups will be removed in favor of internal slots. @@species will be removed in favor of creating `RegExp` objects in the current Realm.
 
-### Constructor
-
-- The `RegExp(` _pattern_, _flags_ `)` constructor will no longer check for `IsRegExp(` _pattern_ `)`.
```

Comment:
for posterity: I was a little surprised that this would be removed. I thought that there would just be a notice that `@@match` is not removed, but just not really used. 

But this is fine. @jandem said that afahk it isn't a hot path.